### PR TITLE
Made documentation better

### PR DESCRIPTION
**Pull request by helveticade**
:+1:

### CODE REVIEW

**Comment by helveticade** [comment]
Yo @daviddahl you'll need to add some stuff to this before we can merge the PR.

File: README.md [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
+-------------
+ddahl to complete.
 
-LICENSE
--------
-See LICENSE file
\ No newline at end of file
+Credits
+-------------
+StatusApp is built by @ddahl, @helveticade, @devgeeks, etc etc at SpiderOak.
```
Comment (daviddahl) [inline README.md]: We can break out a contributors file just to keep this cleaner, no?

**Comment by daviddahl** [comment]
just going to merge this now and work on my changes later

File: README.md (hunk 1 above)
Comment (helveticade) [inline README.md]: Sure. Happy to.

Comment (daviddahl) [inline README.md]: Also we need to add @bennyoak

Comment (helveticade) [inline README.md]: I knew we forgot someone. Sorry @Bennyoak  :<

Comment (Bennyoak) [inline README.md]: Awwe... I feel so loved! :-P

Comment (daviddahl) [inline README.md]: I was just trying to star this thread, LOL

Comment (Bennyoak) [inline README.md]: Seems, instead you **star**ted a thread :-D

**Pull request merged**